Validation/RecoEgamma/plugins/ElectronMcFakePostValidator.cc

```diff
@@ -59,6 +59,14 @@ void ElectronMcFakePostValidator::finalize( DQMStore::IBooker & iBooker, DQMStor
     h1_ele_xOverX0VsEta->setBinContent(ibin,xOverX0) ;
   }
 /**/
+
+  MonitorElement * h1_ele_provenance = get(iGetter, "provenance") ;
+  h1_ele_provenance->getTH1F()->Scale(1./h1_ele_provenance->getBinContent(3));
+  MonitorElement * h1_ele_provenance_barrel = get(iGetter, "provenance_barrel") ;
+  h1_ele_provenance_barrel->getTH1F()->Scale(1./h1_ele_provenance_barrel->getBinContent(3));
+  MonitorElement * h1_ele_provenance_endcaps = get(iGetter, "provenance_endcaps") ;
+  h1_ele_provenance_endcaps->getTH1F()->Scale(1./h1_ele_provenance_endcaps->getBinContent(3));
```

Comment:
Hi Slava,

as a matter of fact, we usually run our tests on as much stat as possible, we were not aware of this use case, sorry.

Can you please confirm that in the low stat test that you mention the histograms are simply empty ? In principle, the bin3 should always be filled as soon as there is an electron candidate. 
If it is indeed the case, we will simply skip the rescaling when the 3rd bin content is empty and it won't matter because the histogram will be empty anyway. I would not feel comfortable to apply different normalization in different cases. 
Thanks